dotnet/src/SemanticKernel/AI/OpenAI/Clients/OpenAIClientAbstract.cs

```diff
@@ -34,6 +35,11 @@ public abstract class OpenAIClientAbstract : IDisposable
 
     private readonly HttpClientHandler _httpClientHandler;
 
+    /// <summary>
+    /// The default timeout for the http client if the caller doesn't supply one.
+    /// </summary>
+    public const int DEFAULT_HTTP_TIMEOUT_IN_SECONDS = 100;
```

Comment:
Remove the underscores from member name Microsoft.SemanticKernel.AI.OpenAI.Clients.OpenAIClientAbstract.DEFAULT_HTTP_TIMEOUT_IN_SECONDS

DefaultHttpTimeoutInSeconds

( I agree this might feel odd, not what I'm used to either )